src/test/kotlin/graphics/scenery/tests/unit/RibbonDiagramTests.kt

```diff
@@ -169,6 +172,19 @@ class RibbonDiagramTests {
         assertEquals(residuesCount, allResidues.size)
     }
 
+    /**
+     * Checks that the important parts of proteins remain unchanged during ribbon diagram calculation
```

Comment:
Why should they change?

Reply:
This was something I checked during debugging the `RibbonDiagram`, the chain of amino acids in the BioJava class is mutable. Since we use this list in our code in multiple ways, I figured it'd be a good idea to verify this.